CONTRIBUTING.md

```diff
@@ -0,0 +1,142 @@
+
+## Governance Model: Salesforce Sponsored
+
+The intent and goal of open sourcing this project is to increase the contributor and user base. However, only Salesforce employees will be given `admin` rights and will be the final arbitrars of what contributions are accepted or not.
+
+## Getting started
+
+Please feel free to join the [Heroku Cloud Native Buildpacks discussions][discussions]. You may also wish to take a look at [Heroku's product roadmap][roadmap] to see where are headed.
+
+## Ideas and Feedback
+
+The `heroku/ruby` buildpack is user centered software focused on application developers. To improve the application developer experience, we need to get real world feedback. We are interested in failure stories "what could be better," success stories "what went well," and your experiences in general.
+
+Please use [Heroku Cloud Native Buildpacks discussions][discussions] to provide feedback, request enhancements, or discuss ideas.
+
+## Issues, Feature Requests, and Bug Reports
+
+Issues should be for bugs in buildpack behavior, and not for general "my application has a problem" support. You can share your experiences building with the heroku/ruby CNB on our [discussions][discussions]. If you're unsure about whether or not to file an issue, please review the [application contract](docs/application_contract.md).
+
+Issues, feature requests, and bug reports are tracked via [GitHub issues on this repository][issues]. If you find an issue and/or bug, please search the issues, and if it isn't already tracked, create a new issue. Please describe the expected behavior, actual behavior, and why the two did not align. If possible, include an [example application that reproduces the problem](https://www.codetriage.com/example_app).
+
+## Fixes, Improvements, and Patches
+
+Fixes, improvements, and patches all happen via [GitHub Pull Requests on this repository][pulls]. If you'd like to improve the tests, you want to make the documentation clearer, you have an alternative implementation of something that may have advantages over the way its currently done, or you have any other change, we would be happy to hear about it. For trivial changes, send a pull request. For non-trivial changes, consider [opening an issue](#issues-feature-requests-and-bug-reports) to discuss it first instead.
+
+## Development
+
+### Dependencies
+
+This buildpack relies on [heroku/libcnb.rs][libcnb] to compile buildpacks. All [libcnb.rs dependencies][libcnb-deps] will need to be setup prior to building or testing this buildpack.
+
+1. Install [rust by following instructions on their site](https://www.rust-lang.org/tools/install)
+1. Follow the setup instructions on [heroku/libcnb](https://github.com/heroku/libcnb.rs)
+1. Install the [pack cli](https://buildpacks.io/docs/for-platform-operators/how-to/integrate-ci/pack/)
```

Comment:
Pack is already covered by the previous link: https://github.com/heroku/libcnb.rs, and in the README.md. Additionally, the url (https://buildpacks.io/docs/for-platform-operators/how-to/integrate-ci/pack/) just changed last week, so maybe we could leave this out and update in fewer places?